Header header
uint8 gpOutStatus

